compiler+runtime/bin/compile

```diff
@@ -2,4 +2,6 @@
 
 set -euo pipefail
 
+git_root=$(git rev-parse --show-toplevel)
+"$git_root"/compiler+runtime/bin/format
```

Comment:
Please use a pre-commit hook for this instead. Nobody else needs this hooked into the compile script; our editors are running the formatter for us.

Reply:
removed.